source/reference/methods.txt

```diff
@@ -876,3 +876,99 @@ Role Management Methods
    * - :method:`db.updateRole()`
 
      - Updates a user-defined role.
+
+Replication Methods 
+-------------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+
+     - Description
+ 
+   * - ``rs.printSecondaryReplicationInfo``
+
+     - Returns the status of the secondary members of the replica set. 
+     
+       This is identical to the ``db.printSecondaryReplicationInfo()`` 
+       method. This method returns the same output as the 
+       :method:`rs.printSlaveReplicationInfo()` method in the legacy 
+       :binary:`~mongo` shell. The :method:`rs.printSlaveReplicationInfo()` method is deprecated in ``mongosh``. Use 
+       ``rs.printSecondaryReplicationInfo()`` instead.
+
+       The following is an example output from the 
+       ``rs.printSecondaryReplicationInfo()`` method issued on a replica set with two secondary members:
+
+       .. code-block:: sh 
+          :copyable: false 
+
+          source: rs2.example.net:27017
+          {
+            syncedTo: 'Tue Oct 13 2020 09:42:18 GMT-0700 (Pacific Daylight Time)',
+            replLag: '0 secs (0 hrs) behind the primary '
+          }
+          ---
+          source: rs3.example.net:27017
+          {
+            syncedTo: 'Tue Oct 13 2020 09:42:18 GMT-0700 (Pacific Daylight Time)',
+            replLag: '0 secs (0 hrs) behind the primary '
+          }
+
+   * - ``db.printSecondaryReplicationInfo``
+
+     - Returns the status of the secondary members of the replica set. 
+     
+       This is identical to the ``rs.printSecondaryReplicationInfo()`` method. 
+       This method returns the same output as the 
+       :method:`db.printSlaveReplicationInfo()` method. The :method:`db.printSlaveReplicationInfo()` method is deprecated in ``mongosh``. Use 
```

Comment:
Ditto here for the removing the `method` link around the second instance of `db.printSlaveReplicationInfo()`. Also needs a line break here.